pkg/scopes/scopes.go

```diff
@@ -19,17 +19,17 @@ func GetScopedTopics(scope, appID string, metadata map[string]string) []string {
 		apps := strings.Split(val, appsSeperator)
 		for _, a := range apps {
 			appTopics := strings.Split(a, appSeperator)
-			if len(appTopics) > 1 {
-				app := appTopics[0]
-				if app != appID {
-					continue
-				}
+			if len(appTopics) == 0 {
+				continue
```

Comment:
In simplifying loop , seems like logic is changed here , as loop is no more breaking when len is 0 compared to earlier version..

Reply:
That's true, the latest logic is correct.